install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```
@@ -107,10 +125,14 @@ spec:
               value: "true"
 {{- end }}
           failureThreshold: 10
+{{- if semverCompare "<1.20-0" .Capabilities.KubeVersion.Version }}
           # The initial delay for the liveness probe is intentionally large to
           # avoid an endless kill & restart cycle if in the event that the initial
           # bootstrapping takes longer than expected.
+          # Starting from Kubernetes 1.20, we are using startupProbe instead
+          # of this field
```

```suggestion
          # Starting from Kubernetes 1.20, we are using startupProbe instead
          # of this field.
```

Just to match with the rest of the formatting in the comment. A few occurrences of this.

---

Fair point, fixed now.